cookbook/integrations/aws/athena/requirements.in

```diff
@@ -0,0 +1,3 @@
+-r ../../../common/requirements-common.in
+
+flytekitplugins-athena>=0.19.0b5
```

Comment:
Do we need the beta version?

Reply:
yes there's no official release out yet, but i can wait to merge this until after we do monthly release (flytekit included)